packages/core/src/vars.css

```diff
@@ -1,6 +1,5 @@
 :root {
 	/* Units........................................................ */
-
```

Comment:
Was this reformat necessary?

Reply:
I just run format in my IDE.

Reply:
Maybe we need to ad more stuff to the .editorconfig file for the future